main.js

```diff
@@ -118,12 +118,17 @@ var App = {
 		return this.buildResult(requestQuery, symbol, engineId);
   },
 
+	// check whether URL starts with a scheme
+	checkUrl(url) {
+		return url.startsWith("//") || url.includes("://");
+	},
+
 	openUrl(url) {
 		// replace history state
 		// so after transition, clicking the back button does not hit find/?q=search
 		// that would transition again to a search result
-		window.history.replaceState({}, 'Find!', window.location.pathname);
-		window.open(url, '_self');
+		if (!this.checkUrl(url)) url = "//" + url;
```

Comment:
I'm not sure about what this does?

Reply:
This is the full qualification I was talking about. If it’s something like `github.com`, this will prepend `//` to get `//github.com`, which is fully qualified.

This is the second point on the list.